code/backend/batch/utilities/helpers/env_helper.py

```diff
@@ -114,6 +114,9 @@ def __load_config(self, **kwargs) -> None:
         self.USE_ADVANCED_IMAGE_PROCESSING = self.get_env_var_bool(
             "USE_ADVANCED_IMAGE_PROCESSING", "False"
         )
+        self.ADVANCED_IMAGE_PROCESSING_MAX_IMAGES = int(
```

Comment:
nit: I see there are helpers e.g. `get_env_var_bool` and `get_env_var_float`, I wondered if it was worth adding `get_env_var_int` for consistency

Reply:
Good point, added